Lib/urllib/parse.py

```diff
@@ -884,6 +885,11 @@ def quote_plus(string, safe='', encoding=None, errors=None):
     string = quote(string, safe + space, encoding, errors)
     return string.replace(' ', '+')
 
+# Expectation: A typical program is unlikely to create more than 5 of these.
```

Comment:
Again, I would go with the default size.  The LRU cache isn't presized; rather, it grows as needed.  So, there is no benefit in selecting a smaller size because of the expected number of cases.  The purpose of the *maxsize* to prevent excessive memory use in the extreme cases.  So we choose the size mainly based on our tolerance worst case.  Since the size of the inputs and outputs for a quoter are very small, we could tolerate thousands of entries and never notice the memory consumption.